src/ProjectTemplates/Web.ProjectTemplates/content/RazorClassLibrary-CSharp/.template.config/it-IT/strings.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": "1.0.0.0",
   "strings": {
-    "name": "Libreria di classi Razor",
+    "name": "Razor Class Library",
```

Comment:
@phenning did these files lose their translation or is this the more correct representation?

Reply:
We may not have gotten all the strings back yet.. remember these were newly onboarded, so the localization vendors will be adding the strings as they get to them, but it won't hold up other translated strings.